index.js

```diff
@@ -0,0 +1,62 @@
+$(document).ready(function(){
+//delaying execution of fucntions until DOM assets are loaded 
+  // indent your code plz :)
+console.log('I got here');
+
+// var city = input;
+
+//reading the contents of the submit form 
+// $('#submit-btn').val(submit);
+
+$('form').submit(function(event) {
+  console.log('got here');
+  
+  
+  // need to preventDefault so form doesnt refresh the page
+  event.preventDefault();
+  
+  console.log('I got to submit btn');
+  // need to extract text from the input with id="city-type"
+  var city = $('#city-type').val();
+  
+
+  // city is your variable
+  //using toLowerCase to normalize 
+  if (city.toLowerCase() === 'nyc' || city.toLowerCase() === 'new york' || city.toLowerCase() === 'new york city') {
+    
+    // $('body').css('.nyc');
+    // use 'addClass'
+    // don't put the '.' when using addClass or removeClass
+    $('body').removeClass('austin sydney la sf');
+    $('body').addClass('nyc');
```

Comment:
This looks great. Just an FYI, since both of these methods/functions `.removeClass()` and `.addClass()` are operating on the same selector `$('body')`, you can actually just chain them:

```js
$('body').removeClass('austin sydney la sf').addClass('nyc');
```